p2p/libp2p/discovery/kadDhtDiscoverer.go

```diff
@@ -77,30 +78,72 @@ func (kdd *KadDhtDiscoverer) Bootstrap() error {
 		return p2p.ErrNilContextProvider
 	}
 
-	ctx := kdd.contextProvider.Context()
-	h := kdd.contextProvider.Host()
+	return kdd.startDHT()
+}
+
+// UpdateRandezVous change the randezVous string, and restart the discovery with the new protocols
+func (kdd *KadDhtDiscoverer) UpdateRandezVous(s string) error {
+	kdd.mutKadDht.Lock()
+	defer kdd.mutKadDht.Unlock()
+
+	if s == kdd.randezVous {
+		return nil
+	}
+
+	err := kdd.stopDHT()
+	if err != nil {
+		log.Warn("Error wile stopDHT, ignore", "err", err)
```

Comment:
Should rephrase this as to be consistent with the rest of the project:
```GO
log.Debug("error wile stopping kad-dht discovery, ignoring", "error", err)
```
Also, this should be on debug level, no one cares if we had an error on kad dht